src/api/app/assets/stylesheets/webui2/comments.scss

```diff
@@ -0,0 +1,22 @@
+.comment-area {
+  width: 100%;
+  margin-bottom: 1rem;
+}
+
+.thread_level_1{
+  margin-left: 1.5em;
+}
+
+.thread_level_2{
+  margin-left: 3em;
+}
+
+.thread_level_3{
+  margin-left: 4.5em;
+}
+
+.thread_level_4{
+  margin-left: 6em;
+}
+
+.hidden { display: none; }
```

Comment:
There is already a Bootstrap class for this [.d-none](https://getbootstrap.com/docs/4.1/utilities/display/)